praxis/layers/fp8_ops.py

```diff
@@ -0,0 +1,202 @@
+# coding=utf-8
```

Comment:
Would it make sense to call the file 

My proposal is to move this file to:
praxis/layers/injection/transformer_engine.py
or 
praxis/layers/injection/fp8_nvidia_gpu.py 

/injection/ because quantization is already taken and works entirely on layer forking.
and not praxis/layers/injection/fp8_quantization.py because it is one particular implementation specific to NVidia GPUs.

WDYT?

Reply:
Let's do the `praxis/layers/injection/fp8_nvidia_gpu.py` because we don't want to confuse the transformer_engine concept in [here](https://github.com/NVIDIA/TransformerEngine).